guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -12,8 +12,22 @@ endif::[]
 ifdef::satellite[]
 * Review Known Issues before you begin an upgrade.
 For more information, see {ReleaseNotesURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+
+.Prerequisites for Disconnected Environment
+If you run {Project} or {SmartProxy} in a disconnected environment, ensure it meets the following requirements:
+
+* You must obtain and deploy Leapp metadata manually.
+For more information, see https://access.redhat.com/articles/3664871[Leapp utility metadata in-place upgrades of RHEL for disconnected upgrades].
+* You have access to {RHEL} and {Project} or {SmartProxy} packages.
+* You must obtain the ISO files for {RHEL} 8 and {Project} or {SmartProxy}.
+For more information, see xref:upgrading_a_disconnected_satellite[].
+* For more information on customizing the Leapp upgrade for your environment, see https://access.redhat.com/articles/4977891[Customizing your {RHEL} in-place upgrade].
+* Since Leapp completes part of the upgrade in a container that has no access to additional ISO mounts, the repositories cannot be served from a locally mounted ISO but must be delivered over the network from a different machine.
+* For more information, see https://access.redhat.com/solutions/5492401[How to in-place upgrade an offline / disconnected RHEL 7 machine to RHEL 8 with Leapp?]
 endif::[]
+ifndef::satellite[]
 * Access to available repositories or a local mirror of repositories.
+endif::[]
 * If you previously upgraded {Project} or {SmartProxy} from an earlier version, and the `/var/lib/pgsql` contained the PostgreSQL database content before the migration from PostgreSQL 9 to PostgreSQL 12 from the SCL, empty `/var/lib/pgsql` before proceeding.
 * During the upgrade, the PostgreSQL data is moved from `/var/opt/rh/rh-postgresql12/lib/pgsql/data/` to `/var/lib/pgsql/data/`.
 If these two paths reside on the same partition, no further action is required.
```

Comment:
As noted in https://github.com/theforeman/foreman-documentation/pull/1427#discussion_r989823230, those two are currently rendered under "disconnected":
![image](https://user-images.githubusercontent.com/94284/195817114-6d518b48-16b0-4bb5-8d4b-46e37cf238d7.png)
However, the PostgreSQL information applies equally to both connected and disconnected usescases.